src/vcConvert.cpp

```diff
@@ -518,6 +516,7 @@ void vcConvert_ShowUI(vcState *pProgramState)
       if (ImGui::InputText("##vcSetTemporaryDirectoryText", tempDirectory, udLengthOf(tempDirectory)))
         vdkConvert_SetTempDirectory(pSelectedJob->pConvertContext, tempDirectory);
 
+      /*
```

Comment:
could you either remove the contents within this comment, or put a note describing why it's been commented out and left here